source/extensions/io_socket/user_space/file_event_impl.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include <cstdint>
+
+#include "envoy/event/file_event.h"
+
+#include "common/common/assert.h"
+#include "common/event/dispatcher_impl.h"
+#include "common/event/event_impl_base.h"
```

Comment:
Is this used? I didn't see anything clearly using it.

Reply:
Yeah... event_impl_base is tied to libevent raw event. Stripping